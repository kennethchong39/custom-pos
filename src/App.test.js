import renderer from 'react-test-renderer';

import App from './App';

test('component: App tests', () => {
  const tree = renderer.create(<App />).toJSON();

  expect(tree).toMatchSnapshot();
});
