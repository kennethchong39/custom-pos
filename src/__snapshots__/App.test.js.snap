// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component: App tests 1`] = `
<div
  className="container"
>
  <div
    className="header"
  >
    <h1>
      <span
        className="deliverr"
      >
        Deliverr
      </span>
       Caf√©
    </h1>
  </div>
  <div
    className="body"
  >
    <div
      className="menu"
    >
      <div
        className="menu-title"
      >
        <h2>
          Menu
        </h2>
      </div>
      <hr />
      <div
        className="menu-items"
      />
    </div>
    <div
      className="current-order"
    >
      <div
        className="current-order-header"
      >
        <h2>
          Cart
        </h2>
        <hr />
      </div>
      <div
        className="current-orders"
      >
        <div
          className="current-item-empty"
        >
          <p>
            Empty Cart
          </p>
        </div>
      </div>
      <div
        className="current-order-footer"
      >
        <div
          className="current-order-total"
        >
          <p>
            Total
          </p>
          <p>
            $ 
            0
          </p>
        </div>
        <button
          className="disabled"
          disabled={true}
          onClick={[Function]}
        >
          Submit Order
        </button>
      </div>
    </div>
    <div
      className="active-order"
    >
      <div
        className="active-order-header"
      >
        <h2>
          All Orders
        </h2>
        <hr />
      </div>
    </div>
  </div>
</div>
`;
